docs/local-backend-roles/localroles.md

```diff
@@ -0,0 +1,30 @@
+# Setting roles in your local backend
+
+Please make sure that you have setup the backend locally. Watch these videos if you have not done that: 

```

Comment:
```suggestion
Please make sure that you have setup the backend locally by running the command `yarn validate-setup`. Watch these videos if you have not done that: 
```

Reply:
Or we can add a link to this as well https://github.com/Real-Dev-Squad/website-backend#confirm-correct-configuration-setup